terraform/deployment/modules/base/main.tf

```diff
@@ -0,0 +1,306 @@
+locals {
+  autoscaling_settings = {
```

Comment:
do we need scaling

Reply:
Nope this is only the var declaration if we want to reenable it. We don't pass it to the ecs module. I could remove it though to make it more clear. Will do it in a follow-up PR.